src/CollAction/Controllers/ProjectsController.cs

```diff
@@ -204,6 +212,28 @@ public ViewResult StartInfo()
                 return View(editProjectViewModel);
             }
 
+            VideoLink descriptionVideoLink = null;
+
+            // If the video link hasn't changed...
+            if (project.DescriptionVideoLink?.Link == editProjectViewModel.DescriptionVideoLink)
```

Comment:
Could this be extracted somewhere else other than the controller? It seems like this logic should belong to a model layer somewhere.